Example/PageBuilderExtensionSlider/etc/di.xml

```diff
@@ -0,0 +1,15 @@
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\PageBuilder\Model\WidgetInitializerConfig">
+        <arguments>
+            <argument name="config" xsi:type="array">
+                <item name="slider" xsi:type="array">
+                    <item name="default" xsi:type="array">
+                        <item name="component" xsi:type="string">Example_PageBuilderExtensionSlider/js/content-type/slider/appearance/default/widget</item>
+                        <!--optional if you want to load widget per appearance-->
```

Comment:
Why is this optional? I thought the appearance name would be the merge key for this configuration

Reply:
True. Removed.